electron/main/llm/llmConfig.ts

```diff
@@ -33,3 +32,77 @@ export function validateAIModelConfig(
 
   return null;
 }
+
+export async function addOrUpdateLLMSchemaInStore(
```

Comment:
Not particular sure about this, but is there an `add model` for ollama needed here?

I see that there is a `deleteModel` function call for Ollama in the removeLM function below.

Reply:
For now we use the separate function pullModel to attach ollama models. They are in different routes because pullModel will send back the download status to the frontend